membership-attribute-service/app/models/AccountDetails.scala

```diff
@@ -214,13 +214,13 @@ object AccountDetails {
   }
   def mmaCategoryFrom(product: Product): String = product match {
     case _: Product.Paper => "subscriptions" // Paper includes GW 🤦‍
```

Comment:
Should this line have been converted too?

Reply:
no, good spot but paper is a trait

Reply:
see
https://github.com/guardian/members-data-api/blob/d80b77bac1af42560a40299dbdd511cadcb418f4/membership-common/src/main/scala/com/gu/memsub/ProductFamily.scala#L17